js/tji-datasets-explore.js

```diff
@@ -318,24 +321,45 @@ TJIChartView.prototype.create_filter_panel = function() {
 
   var $filters = jQuery('<form id="js-TJIfilters" />');
   _.each(this.filters, function(f) {
-    var fieldset = jQuery('<fieldset><legend>' + f.key.replace(/_/g, " ") + '</legend></fieldset>');
+    var fieldset = jQuery('<fieldset><legend class="legend">' + f.key.replace(/_/g, " ") + '</legend></fieldset>');
+    var filterset = jQuery('<div class="filter-set"><div class="filter"><input type="checkbox" class="toggle" name="toggle"><label for="toggle">Select/Unselect All</label></div></div>');
```

Comment:
Hey Simi, do you mean that instead of making the toggle and input I should make it a link instead? I can make it so that if any of the filter boxes are unchecked, the toggle button will also be unchecked. That way, "unselect all" will only happen if all the boxes are checked.